src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -145,3 +145,114 @@
 type: command
 short-summary: List all the databases and its collections that can be restored in the given account at the given timesamp and region.
 """
+
+helps['cosmosdb sql role'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL role resources.
+"""
+
+helps['cosmosdb sql role definition'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL role definitions.
+"""
+
+helps['cosmosdb sql role definition create'] = """
+type: command
+short-summary: Create a SQL role definition under an Azure Cosmos DB account.
+examples:
+  - name: Create a SQL role definition under an Azure Cosmos DB account.
+    text: az cosmosdb sql role definition create -a MyAccount -g MyResourceGroup -b @role-definition.json
```

Comment:
could you also provide examples for json string like this?
![image](https://user-images.githubusercontent.com/49508232/106427124-c3a7d300-64a1-11eb-91bd-c7b142aa34a7.png)